### ISSUE TITLE
OSX Cannot launch "DIYLC", No compatible version of Java 1.8+ available

### ISSUE THREAD

**Issue by MikeMitterer** [bancika/diy-layout-creator#383]
```bash
echo $JAVA_HOME
/Library/Java/JavaVirtualMachines/jdk1.8.0_151.jdk/Contents/Home
```
But... the real strange thing is that if I right click `diylc.jar` and if I say "Open with Jar launcher" it opens and it works...

**Comment by leighmarble** [comment]
Same behavior here, on Mac OS 10.11.6.

**Comment by bancika** [comment]
Fixed in 3.44.1, please update

**Comment by leighmarble** [comment]
Great, that fixed it for me.
